src/property-filter/token-editor.tsx

```diff
@@ -199,21 +205,33 @@ export function TokenEditor({
       )}
 
       <div className={styles['token-editor-actions']}>
-        <InternalButton
-          formAction="none"
-          variant="link"
-          className={clsx(styles['token-editor-cancel'], testUtilStyles['token-editor-cancel'])}
-          onClick={onDismiss}
+        <span
+          {...getAnalyticsMetadataAttribute({
```

Comment:
Can we pass this as an internal prop to the button, similar to how it is done for popover?

Reply:
I prefer as mush as possible to avoid adding new properties to components, if there is another solution. Is there any risk in adding this span?

Reply:
The exception is rather the Popover. I didn't manage to avoid it there because wrapping the whole popover in another element with the metadata would lead to overriding the `action` of every single event coming from the popover, instead of the close action only.